clusters/threat-actor.json

```diff
@@ -504,7 +514,10 @@
           "https://web.archive.org/web/20130920000343/https://www.symantec.com/connect/blogs/hidden-lynx-professional-hackers-hire",
           "https://www.recordedfuture.com/hidden-lynx-analysis/",
           "https://www.secureworks.com/research/threat-profiles/bronze-keystone",
-          "https://attack.mitre.org/groups/G0025/"
+          "https://attack.mitre.org/groups/G0025/",
+          "cfr.org/cyber-operations/axiom",
+          "https://attack.mitre.org/groups/G0001/",
+          "https://www.youtube.com/watch?v=NFJqD-LcpIg"
         ],
         "synonyms": [
           "APT 17",
```

Comment:
```suggestion
```

Reply:
We already have APT17 in synonyms.
Also, remove "Deputy Dog" from synonyms as it's a malware used by them.

Reply:
Any specific reference concerning the fact that DeputyDog/Deputy Dog is more tooling than a TA or activity group? I remember FireEye mentioned it as a campaign at some point.

https://paper.seebug.org/papers/APT/APT_CyberCriminal_Campagin/2013/Operation_EphemeralHydra.pdf

Reply:
Here from cisco https://blogs.cisco.com/security/talos/threat-spotlight-group-72

Besides, people used to name campaign after malware families which is the case for FE.

Reply:
Yep this is kind of a challenge (we might need to improve the `tool` galaxy at some point to reflect the confusion).  Thanks a lot for the effort @r0ny123  and @Mathieu4141 - I'll merge it.